src/current/_includes/releases/v24.2/v24.2.5.md

```diff
@@ -0,0 +1,118 @@
+- Fixed a bug introduced before v23.1 that could cause a composite sensitive expression to compare differently if supplied equivalent but not identical input values, such as `2.0::DECIMAL` and `2.00::DECIMAL`. : which could lead to incorrect results in rare cases. The join must lookup into a table with a computed index column where the computed column expression is composite sensitive. [#132508][#132508]
+- Fixed a bug where a span statistics request on a mixed-version cluster could result in an null pointer exception. [#132680][#132680]
+- Updated the `franz-go` library to fix a potential deadlock when a changefeed restarts. [#132787][#132787]
+- Fixed a bug where a changefeed could fail to update protected timestamp records after a retryable error. [#132773][#132773]
+- Fixed a bug where a changefeed that used change data capture queries could fail after a system table was garbage collected. [#131649][#131649]
+- Fixed a rare bug introduced in v22.2 where an update of a primary key column could fail to update the primary index if it is also the only column in a separate column family. [#132120][#132120]
+- Fixed a bug where the `proretset` column of the `pg_catalog.pg_proc` table was incorrectly set to `false` for builtin functions that return a set. [#132876][#132876]
+- Fixed a bug that could cause incorrect evaluation of scalar expressions with `NULL` values. [#132946][#132946]
+- Fixed a rare bug in the query optimizer that could cause a node to crash if a query contained a filter in the form `col IN (elem0, elem1, ..., elemN)` when `N` is very large, in the order of millions, and when `col` exists in a hash-sharded index or when a table with an indexed computed column depends on `col`. [#132868][#132868]
+- Fixed a bug where an `ALTER DEFAULT PRIVILEGES FOR target_role ...` command could result in an erroneous privilege error when run by a with the `admin` role. [#133071][#133071]
+- Fixed a bug where a `REASSIGN OWNED BY` command would fail to transfer ownership of the public schema even if it was owned by the target role. [#133071][#133071]
+- Fixed a panic when resolving the types of an `AS OF SYSTEM TIME` expression. [#132455][#132455]
+- Fixed a bug that could cause new connections to fail with the following error after upgrading: `ERROR: invalid value for parameter "vectorize": "unknown(1)" SQLSTATE: 22023 HINT: Available values: off,on,experimental_always`. To encounter this bug, the cluster must have:
+    1. Run on version v21.1 at some point in the past
+    1. Run `SET CLUSTER SETTING sql.defaults.vectorize = 'on';` while running v21.1.
+    1. **Not** set `sql.defaults.vectorize` after upgrading past v21.1 4.
+    1. Subsequently upgraded to v24.2.upgraded all the way to v24.2.
```

Comment:
```suggestion
    1. Subsequently upgraded to v24.2, after all previous required versions.
```

Reply:
Updated your comment to remove the fragment I introduced, but there were only required releases before v24.2, so the other part of the edit didn't make sense to me.